secure_message/common/alerts.py

```diff
@@ -48,5 +48,5 @@ def __init__(self, alerter=None):
         if alerter is not None:
             self.alert_method = alerter
 
```

Comment:
Comment only- No change needed: With only 2 send methods duplicating the logging is ok ish. However since the AlertViaLogging and AlertViaGovNotify are only plug ins for the AlertUser class we could log in AlertUser.send and then only log the send method in the plug in classes .  This would men that we would not need to pass params to the plug in classes simply for the logs .